src/stat.js

```diff
@@ -127,3 +127,111 @@ export function varianceAll(matrix, unbiased, mean) {
     return (sum2 - (sum1 * sum1) / size) / size;
   }
 }
+
+
+/**
+ * Calculate standard deviation column by column of a matrix
+ * @param {Matrix} matrix - Column index
+ * @return {Array}
+ */
+export function stdDeviationByColumn(matrix) {
+  let stdDeviationByColumn = [];
+  let sum = 0;
+  let mean = [];
+  for (let j = 0; j < matrix.columns; j++) {
+    for (let i = 0; i < matrix.rows; i++) {
+      sum += matrix.get(i, j);
+    }
+    mean[j] = (sum / matrix.rows);
+    sum = 0;
+  }
+  for (let j = 0; j < matrix.columns; j++) {
+    for (let i = 0; i < matrix.rows; i++) {
+      sum += Math.pow(matrix.get(i, j) - mean[j], 2);
+    }
+    stdDeviationByColumn[j] = Math.sqrt(sum / (matrix.rows - 1));
+    sum = 0;
+  }
+  return stdDeviationByColumn;
+}
+
+/**
+ * Calculate the covariance of two matrix
+ * @param {Matrix} xMatrix
+ * @param {Matrix} yMatrix
+ * @param {object} [options]
+ * @param {boolean} [options.center]
+ * @return {Array}
+ */
+export function cov(xMatrix, yMatrix, options = {}) {
```

Comment:
I think we should use the full name for clarity:
```suggestion
export function covariance(xMatrix, yMatrix, options = {}) {
```